x/pki/client/cli/query_pki_revocation_distribution_points_by_issuer_subject_key_id.go

```diff
@@ -0,0 +1,39 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+	pkitypes "github.com/zigbee-alliance/distributed-compliance-ledger/types/pki"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/utils/cli"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/x/pki/types"
+)
+
+func CmdShowPkiRevocationDistributionPointsByIssuerSubjectKeyID() *cobra.Command {
+	var issuerSubjectKeyID string
+
+	cmd := &cobra.Command{
+		Use:   "revocation-points-by-issuer",
```

Comment:
Can we name it just `revocation-points`? If not, then let's call this one `revocation-points`, and rename `revocation-points` from CmdListPkiRevocationDistributionPoint to `all-revocation-points`.